diffkemp/simpll/library/FFI.h

```diff
@@ -0,0 +1,121 @@
+//===------------------ FFI.h - C interface for SimpLL --------------------===//
+//
+//       SimpLL - Program simplifier for analysis of semantic difference      //
+//
+// This file is published under Apache 2.0 license. See LICENSE for details.
+// Author: Tomas Glozar, tglozar@gmail.com
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file contains declarations of C functions and structure types used for
+/// interacting with DiffKemp.
+///
+//===----------------------------------------------------------------------===//
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// CFFI_DECLARATIONS_START
```

Comment:
Add a comment (probably above this one) that this line must not be altered nor deleted.